templates/nginx.j2

```diff
@@ -27,6 +27,13 @@ server {
     add_header Strict-Transport-Security "max-age=31536000";
     add_header X-Content-Type-Options nosniff;
 
+    gzip {{ openwisp2_nginx_ssl_config.gzip }};
+    gzip_comp_level {{ openwisp2_nginx_ssl_config.gzip }};
+    gzip_proxied {{ openwisp2_nginx_ssl_config.gzip }};
+    gzip_min_length {{ openwisp2_nginx_ssl_config.gzip }};
+    gzip_proxied {{ openwisp2_nginx_ssl_config.gzip }};
+    gzip_types {{ openwisp2_nginx_ssl_config.gzip }};
```

Comment:
all these lines refer to `gzip` while they should refer their respective correct key